sentry-ruby/spec/sentry/client/event_sending_spec.rb

```diff
@@ -224,6 +224,15 @@
           expect(event["tags"]["called"]).to eq(true)
         end
       end
+
+      it "doesn't apply before_send_transaction to Event" do
+        dbl = double("before_send_transaction")
+        allow(dbl).to receive(:call)
+        configuration.before_send_transaction = dbl
+
+        expect(dbl).not_to receive(:call)
+        subject.send_event(event)
```

Comment:
Can we avoid using double/mocking when it's not too hard to test it end-to-end? In my experience taking over the old SDK it makes things hard to maintain in the long-term.

Reply:
I did this because of the new codecov enforcing (which was turned on org-wide on github), it was complaining about a test line not being run. I will generally not use doubles, but in this case it makes sense.